### PR TITLE
Live rebuild

### PR DESCRIPTION
**Pull request by sbrunner**
Be able to develop in auto-rebuild mode, a modification will be available in 15s. Instead of more than 10 minutes.

### CODE REVIEW

File: core/Dockerfile [hunk 1]
```diff
@@ -58,4 +58,10 @@ FROM runner AS tester
 COPY jettyRunExtraFiles/mapfish-spring-application-context-override-acceptencetests.xml \
   webapps/ROOT/WEB-INF/classes/mapfish-spring-application-context-override.xml
 
+FROM runner AS watcher
+
+RUN apt update && \
+    apt install --yes python3-pip rsync
```
Comment (danduk82) [inline core/Dockerfile:64]: missing cleanup of apt mess

**Review by danduk82 — CHANGES_REQUESTED** [review]

File: core/Dockerfile (hunk 1 above)
Comment (sbrunner) [reply core/Dockerfile:64]: For me, we don't need to do that on an image that used only locally.
If you don't agree, no problem to add that...

Comment (danduk82) [reply core/Dockerfile:64]: it still uses more place on the HD. Yesterday I cleaned 56Gb with a `docker system prune`

Comment (sbrunner) [reply core/Dockerfile]: Done :-)

**Review by danduk82 — APPROVED** [review]
btw, this is VERY nice feature, thanks

**Pull request merged**